webapp/greenhouse.py

```diff
@@ -123,7 +122,7 @@ def to_dict(self):
             "date": self.date,
             "featured": self.featured,
             "fast_track": self.fast_track,
-            "departments": sector,
+            "departments": [dept.name for dept in self.departments],
```

Comment:
This line is the main part of this fix - turn `Vacancy.departments` into a list rather than a string.